library/ssl_tls.c

```diff
@@ -1953,15 +1949,19 @@ void mbedtls_ssl_set_verify(mbedtls_ssl_context *ssl,
 #if defined(MBEDTLS_KEY_EXCHANGE_ECJPAKE_ENABLED)
 
 #if defined(MBEDTLS_USE_PSA_CRYPTO)
+/* The only two JPAKE user/peer identifiers supported for the time being. */
```

Comment:
When it comes to TLS, that comment is not accurate. That's the two strings specified by https://datatracker.ietf.org/doc/html/draft-cragie-tls-ecjpake-01 and as such the only we'll ever use here.